src/irclog2html/logs2html.py

```diff
@@ -231,6 +234,14 @@ def process(dir, options):
         extra_args += ['-S']
     if options.dircproxy:
         extra_args += ['--dircproxy']
+    if options.output_dir:
+        if os.path.isdir(options.output_dir):
+            extra_args += ['--output-file', options.output_dir]
+            out_dir = options.output_dir
+        else:
+            raise Error('Argument specified in --output-file is not a directory')
+    else:
+        out_dir = dir
```

Comment:
Perhaps it would make sense to try and create the specified output directory if it's missing (with `os.makedirs(options.output_dir)`)?